crates/network-primitives/src/traits.rs

```diff
@@ -23,6 +27,36 @@ pub trait ReceiptResponse {
 
     /// Number of the block this transaction was included within.
     fn block_number(&self) -> Option<u64>;
+
+    /// Transaction Hash.
+    fn transaction_hash(&self) -> TxHash;
+
+    /// Index within the block.
+    fn transaction_index(&self) -> Option<u64>;
+
+    /// Gas used by this transaction alone.
+    fn gas_used(&self) -> u128;
+
+    /// Effective gas price.
+    fn effective_gas_price(&self) -> u128;
+
+    /// Blob gas used by the eip-4844 transaction.
+    fn blob_gas_used(&self) -> Option<u128>;
+
+    /// Blob gas price paid by the eip-4844 transaction.
+    fn blob_gas_price(&self) -> Option<u128>;
+
+    /// Address of the sender.
+    fn from(&self) -> Address;
+
+    /// Address of the receiver.
+    fn to(&self) -> Option<Address>;
+
+    /// Post-transaction state root.
+    fn state_root(&self) -> Option<B256>;
```

Comment:
we can remove this, this isn't filled anymore afaik

Reply:
fixed